packages/next/src/build/index.ts

```diff
@@ -2741,8 +2737,9 @@ export default async function build(
 
             // When this is an app page and PPR is enabled, the route supports
             // partial pre-rendering.
-            const experimentalPPR =
-              !isRouteHandler && config.experimental.ppr === true
+            const experimentalPPR: true | undefined =
```

Comment:
Perhaps this should be a `boolean` rather than `true | undefined`

Reply:
We only add the `experimentalPPR` field here if it's true. If it's `false`, we just don't write it at all. It shouldn't be a persistant boolean, as we expect to deprecate it, so this type here ensures we never write `false` and set a precedent that `experimentalPPR` will always be defined and will be a boolean.